app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   has_many :comments, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
+
+  def load_last_three_posts
+    Post.last(3)
+  end
```

Comment:
- [ ] Optional suggestion

Use this implementation

posts.order('created_at Desc').limit(3)

Reply:
Can you try first rake db:reset after rails db:migrate because when I try everything seems good